src/main/java/com/iota/iri/service/API.java

```diff
@@ -201,14 +225,18 @@ private AbstractResponse process(final String requestString, InetSocketAddress s
             }
 
             if (instance.configuration.string(DefaultConfSettings.REMOTE_LIMIT_API).contains(command) &&
-                    !sourceAddress.getAddress().isLoopbackAddress()) {
+                    !exchange.getSourceAddress().getAddress().isLoopbackAddress()) {
```

Comment:
concatenated gets...

Reply:
I will do that, although the extra variable will be set even when the first condition isn't met. That means we are trading performance for maintainability.